src/controller.js

```diff
@@ -0,0 +1,111 @@
+const fs = require('fs');
+const path = require('path');
+
+const { task1: sort, task2: biggestPrice, task3 } = require('./tasks');
+
+let store = require('./inputData');
+
+function ok(res, body = { message: 'Ok' }) {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = 200;
+  res.write(JSON.stringify(body));
+  res.end();
+}
+
+function badRequest(res, message = { message: 'Bad request!' }) {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = 400;
+  res.write(JSON.stringify(message));
+  res.end();
+}
+
+function methodNotAllowed(res) {
+  res.setHeader('Content-Type', 'application/json');
+  res.statusCode = 405;
+  res.write(JSON.stringify({ message: 'Method not allowed!' }));
+  res.end();
+}
+
+function isIncorrectData(data) {
+  if (!Array.isArray(data) || data.length === 0) return true;
+
+  const incorrectArray = data.filter(
+    value =>
+      typeof value !== 'object' ||
+      !(
+        'type' in value &&
+        'color' in value &&
+        ('price' in value || 'priceForPair' in data[0])
+      ),
+  );
+  if (incorrectArray.length !== 0) return true;
+
+  return false;
+}
+
+module.exports.task1 = (request, response) => {
```

Comment:
There is a high chance of exporting something twice and overwrite the first function.
Keep `module.export` simple. See examples in the comment to `config` file